### PR TITLE
Add beta agreement

### PR DESCRIPTION
**Pull request by mattwiller**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mattwiller has signed the CLA. Thanks for the pull request!

File: docs/usage/getting-started.md [hunk 1]
```diff
@@ -146,3 +146,9 @@ func obtainJWTTokenFromExternalSources() -> DelegatedAuthClosure {
 ```
 
 __Step 5__: Run the sample app
+
+License
+-------
+
+Any use of this software is governed by the attached [Box SDK Beta Agreement](../../BETA-AGREEMENT.md).  If you do not
```
Comment (PJSimon) [inline docs/usage/getting-started.md:153]:
```suggestion
Any use of this software is governed by the attached [Box SDK Beta Agreement](../../BETA-AGREEMENT.md).  By using this software,
```

File: docs/usage/getting-started.md [hunk 2]
```diff
@@ -146,3 +146,9 @@ func obtainJWTTokenFromExternalSources() -> DelegatedAuthClosure {
...
+accept the terms of the Box SDK Beta Agreement, you may not use this software.
```
Comment (PJSimon) [inline docs/usage/getting-started.md:154]:
```suggestion
you accept the terms of the Box SDK Beta Agreement.
```

File: docs/usage/getting-started.md (hunk 1 above)
Comment (PJSimon) [reply docs/usage/getting-started.md:153]: After reviewing "Proposed Changes to GitHub page of Box iOS SDK by Legal.docx" ( https://cloud.app.box.com/s/0fe26b12k1ekkrzpimrnnnn19d89x320) ... never mind.

File: docs/usage/getting-started.md (hunk 2 above)
Comment (PJSimon) [reply docs/usage/getting-started.md:154]: After reviewing "Proposed Changes to GitHub page of Box iOS SDK by Legal.docx" ( https://cloud.app.box.com/s/0fe26b12k1ekkrzpimrnnnn19d89x320) ... never mind.

**Review by PJSimon — APPROVED** [review]

**Pull request merged**